apps/estuary/lib/estuary/services/event_retrieval_service.ex

```diff
@@ -2,21 +2,15 @@ defmodule Estuary.Services.EventRetrievalService do
   @moduledoc """
   Interface for retrieving events.
   """
-  alias Estuary.Query.Select
   alias Estuary.Datasets.DatasetSchema
+  alias Estuary.Query.Helper.PrestigeHelper
 
-  def get_all() do
-    make_select_table_schema()
-    |> Select.select_table()
-  end
-
-  defp make_select_table_schema do
-    %{
-      "columns" => ["author", "create_ts", "data", "type"],
-      "table_name" => DatasetSchema.table_name(),
-      "order_by" => "create_ts",
-      "order" => "DESC",
-      "limit" => 1000
-    }
+  def get_all do
+    "SELECT events.author, events.create_ts, events.data, events.type
+    FROM (SELECT author, create_ts, data, type
+    FROM #{DatasetSchema.table_name()}
+    WHERE create_ts >= to_unixtime(now() - interval '3' hour) LIMIT 1000) events
+    ORDER BY events.create_ts DESC"
+    |> PrestigeHelper.execute_query()
```

Comment:
I think this might read better as a multiline string:

```
    query = """
            SELECT events.author, events.create_ts, events.data, events.type
            FROM (SELECT author, create_ts, data, type
            FROM #{DatasetSchema.table_name()}
            WHERE create_ts >= to_unixtime(now() - interval '3' hour) LIMIT 1000) events
            ORDER BY events.create_ts DESC
            """
    PrestigeHelper.execute_query(query)
```